doc/setting_up_vcpkg.md

```diff
@@ -2,6 +2,43 @@
 
 This document describes how to setup vcpkg to build applications using Microsoft Azure IoT device C SDK. It demonstrates building and running a C language SDK sample for Windows, Linux, and Mac.
 
+## C SDK vcpkg support
+
+The C SDK uses [vcpkg](https://github.com/microsoft/vcpkg) primarily for LTS releases. The most recent LTS release can be installed using the syntax: `vcpkg install azure-iot-sdk-c` and builds using the following CMake flags:
+
+```
+    -Dskip_samples=ON
+    -Duse_installed_dependencies=ON
+    -Duse_default_uuid=ON
+    -Dbuild_as_dynamic=OFF
+    -Duse_edge_modules=ON
+    -Dwarnings_as_errors=OFF
+```
+
+The `use-prov-client` feature uses the syntax: `vcpkg install azure-iot-sdk-c[use-prov-client]` and builds using additional CMake flags:
+```
+    -Dhsm_type_symm_key=ON
+    -Duse_prov_client=ON
+```
+
+The `public-preview` feature allows you to build and install the azure-iot-sdk-c from a provided public-preview hash. The Ref hash used can be found in the [portfile.cmake](https://github.com/microsoft/vcpkg/blob/master/ports/azure-iot-sdk-c/portfile.cmake) file. To access this feature, use the syntax: `vcpkg install azure-iot-sdk-c[public-preview]`. The following CMake flags will be set:
+
+```
+    -Dskip_samples=ON
+    -Duse_installed_dependencies=ON
+    -Duse_default_uuid=ON
+    -Dbuild_as_dynamic=OFF
+    -Duse_edge_modules=ON
+    -Dwarnings_as_errors=OFF
+```
+
+There are no other features available.
```

Comment:
Nit - instead of "features" I'd call this something like "There are no other CMake configuration options exposed via vcpkg" since most features you "just get" in the core IoT pkg and not related to cmake settings.

Reply:
"Features" (or [feature packages](https://devblogs.microsoft.com/cppblog/vcpkg-introducing-installation-options-with-feature-packages/#:~:text=Vcpkg%20is%20a%20package%20manager%20to%20help%20acquiring,library%20as%20you%20can%20specify%20different%20options%20%28features%29) is specifically a vcpkg term here for installing optional packages.  But that could easily be confused with C SDK features. I'll change.

From vcpkg.json file azure-iot-sdk-c:
```
"features": {
    "public-preview": {
      "description": "A version of the azure-iot-sdk-c containing public-preview features.",
      "dependencies": [
        {
          "name": "azure-c-shared-utility",
          "features": [
            "public-preview"
          ]
        },
...
      ]
    },
    "use-prov-client": {
      "description": "Enables device provisioning client for DPS"
    }
  }
```